apps/website/.vuepress/theme/enhanceApp.js

```diff
@@ -20,5 +20,12 @@ export default ({
       dsn: 'https://3e41de8eb9e440d6b834fc556ad8e83c@o378402.ingest.sentry.io/5201749',
       integrations: [new VueIntegration({ Vue, attachProps: true, logErrors: true })],
     });
+
+    // Register the service worker after it has everything it needs.
+    if ('serviceWorker' in navigator) {
```

Comment:
I added this here so it happens early in the Vue bootstrap process. 
Not 💯 sure if its the best pace for it.